cirq-ft/cirq_ft/algos/unary_iteration.ipynb

```diff
@@ -261,9 +261,9 @@
    "outputs": [],
    "source": [
     "import cirq\n",
-    "from cirq._compat import cached_property\n",
     "from cirq_ft import Signature, GateWithRegisters\n",
     "from cirq_ft.infra.bit_tools import iter_bits\n",
+    "from functools import cached_property\n",
```

Comment:
Please move this import to the top of the import block as it is an import from a standard library.
Same for the import at line 475.